httpbakery/form/form.go

```diff
@@ -49,11 +50,17 @@ client's responsibility to interpret the schema and present it to the
 user.
 */
 
-// Filler represents an object that can fill out a form. The given schema
-// represents the form template. The returned value should be compatible
-// with this.
+// Form describes a form that needs to be completed to perform the login.
+type Form struct {
```

Comment:
+1 to making this a struct, as I can definitely see the potential for more fields here in the future.